### DISCUSSION TITLE
v1.0.0-dev.4

### DISCUSSION THREAD

**Discussion by clementroche** [darkroomengineering/lenis#102] [Announcements]
- introduce `normalizeWheel` options

<hr /><em>This discussion was created from the release <a href='https://github.com/studio-freight/lenis/releases/tag/v1.0.0-dev.4'>v1.0.0-dev.4</a>.</em>

**Comment by StevenJPx2** [comment]
Can't wait for the official v1! I assume that with virtual-scroll dropped for nanoevents that Lenis will be faster than it's current official release?
